src/components/RecipeCard/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ *
+ * RecipeCard
+ *
+ */
+import PropTypes from 'prop-types';
+import { Card } from 'antd';
+import React from 'react';
+import './RecipeCard.css';
+const { Meta } = Card;
+// import styled from 'styled-components';
+
+function RecipeCard(props) {
+  return (
+    <div className="recipe_card" onClick={props.handleClick}>
+      <Card
+        hoverable
+        style={{ width: 170, borderRadius: 15 }}
+        cover={
+          <img
+            alt={props.name}
+            src={props.imageUrl}
+            style={{
```

Comment:
For better styling (code style), create a const `style` outside the component and use it here.

```js
const styles = {
  img: { … }
}

…

<img style={styles.img} />
```

Reply:
Gotcha